clang/test/SemaCXX/type-traits.cpp

```diff
@@ -2671,212 +2660,212 @@ struct has_unique_object_representations {
   static const bool value = __has_unique_object_representations(T);
 };
```

Comment:
This is no longer needed at all

Reply:
The intent here was to emulate how the library would use the builtin as a type-trait.  

Also, see this bug here: https://github.com/llvm/llvm-project/issues/77585 for additional comments.

Reply:
Except we don't do that for any other trait in that file.

Reply:
Yep, thats true too, it was the intent at the time though (and to better emulate hte test cases we were given, note that the whole section is basically a copy/paste + de-c++17'izing of what STL gave me).

My comment on the bug has more context.